rust-workspace/crates/ntc-data-packages/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "ntc-data-packages"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.56"
```

Comment:
Why did we declare `rust-version = 1.56` for `ntc-data-packages` and `rust-version = 1.57` for `ntc-vault-cli`?

Reply:
That's the minimum support rust version (or MSRV) that each crate compiles with.

`1.56` is the minimum baseline for Rust 2021 edition, so that's essentially the default value for `rust-version`.

**ntc-vault-cli** needs `1.57` for the macro output of **serde_with**: you can try compiling it with 1.56 to see the compiler error. 😄

(I use [cargo-msrv](https://foresterre.github.io/cargo-msrv/) to quickly check and set the MSRV while developing: you can test it locally by setting `rust-version = "1.56"` and running `cargo msrv`.)